workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/schema/QueryInspector.java

```diff
@@ -479,10 +479,11 @@ public CQLFragment visitRelation(RelationContext ctx) {
         if (variable == QUESTION_MARK) {
           variable = INTERNAL_TOKEN_VARNAME;
         }
-        if (ctx.relationType().getText().equals(">")) {
+        if (ctx.relationType().getText().equals(">=") || ctx.relationType().getText().equals(">")) {
```

Comment:
Here in this file : 

> partitioner/src/main/java/com/datastax/oss/dsbulk/partitioner/TokenRangeReadStatementGenerator.java 

There are 2 methods for 'generate' so where-ever the first method is used it will still generate the query with '>' and '<='.

Reply:
Should we not change it in similar way?